src/osm_configurator/view/constants/progress_bar_constants.py

```diff
@@ -11,8 +11,9 @@ class ProgressBarConstants(Enum):
     PROGRESS_BAR_CONSTANTS_BORDER_WITH = 0
     PROGRESS_BAR_CONSTANTS_CORNER_RADIUS = 8
     PROGRESS_BAR_CONSTANTS_FG_COLOR = "000000"
-    PROGRESS_BAR_CONSTANTS_PROGRESS_COLOR = "green"
+    PROGRESS_BAR_CONSTANTS_PROGRESS_COLOR = "40B135"
     PROGRESS_BAR_CONSTANTS_ORIENTATION = "horizontal"
     PROGRESS_BAR_CONSTANTS_MODE = "indeterminate"
     PROGRESS_BAR_CONSTANTS_DETERMINATE_SPEED = 1
     PROGRESS_BAR_CONSTANTS_INDETERMINATE_SPEED = 1
+    PROGRESS_BAR_CONSTANTS_WIDTH = 400
```

Comment:
width and height for elements should not be a constant on its own, either calculated from frame constants or be defined in the frame where the element sits, since not all frames will hold same sized elements